python/lsst/daf/butler/tests/_testRepo.py

```diff
@@ -106,7 +104,7 @@ def makeTestCollection(repo):
     """
     # Create a "random" collection name
     # Speed matters more than cryptographic guarantees
-    collection = "test" + "".join((str(i) for i in np.random.randint(0, 10, size=8)))
+    collection = "test" + "".join((str(i) for i in Random().sample(range(10), 8)))
```

Comment:
Um, isn't this changing a sample with replacement to a sample without replacement?   Given the comment, maybe you don't really care (but one may be faster than the other).

Reply:
I think we just want 8 fairly random digits to use in the collection name without having to keep track of a counter internally. Maybe I'm better off doing a single random number of 8 digits and then zero-padding it in the collection name?

Reply:
Is there a function instead that returns 8 random characters instead? (like is used in tempfiles).